packages/burnside-localproxy/src/proxy/proxy.js

```diff
@@ -19,6 +19,7 @@ const defaultOptions = {
   key: certPath + '/localhost.privkey.pem',
   cert: certPath + '/localhost.cert.pem',
   replaceImages: false,
+  trace: false || process.env.trace,
```

Comment:
What is the purpose of `false` here?  Should this be?
```
trace: process.env.trace,
```

Reply:
or `process.env.trace || false`

Reply:
its mostly just to avoid it being undefined. I think either method would work.

Reply:
Just a copypaste error. Thanks for noticing.